File: FileSharing/FileSharing/ClientApp/src/app/Entity/AccountModel.ts
export class AccountModel {
  constructor(Id: string, Username: string, Email: string, Phone: string,
    JWTToken: string, RegistrationDate: Date, FilesUploaded: number,
    FilesDownloaded: number, TotalSizeProcessed: number, Roles: string[]) {
    this.id = Id;
    this.phone = Phone;
    this.email = Email;
    this.filesDownloaded = FilesDownloaded;
    this.filesUploaded = FilesUploaded;
    this.username = Username;
    this.totalSizeProcessed = TotalSizeProcessed;
    this.jwtToken = JWTToken;
    this.registrationDate = RegistrationDate;
    this.roles = Roles;
  }
  public id: string;
  public username: string;
  public email: string;
  public phone: string;
  public jwtToken: string;
  public registrationDate: Date;
  public filesUploaded: number;
  public filesDownloaded: number;
  public totalSizeProcessed: number;
  public roles: string[];
} 

File: FileSharing/FileSharing.Shared/Models/AccountModel.cs


namespace FileSharing.Shared.Models
{
    public class AccountModel
    {
        public string Id { get; set; } = string.Empty;
        public string Username { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public string JWTToken { get; set; } = string.Empty;
        public DateTime RegistrationDate { get; set; } = DateTime.Now;
        public string[]? Roles { get; set; }
    }
}
